src/components/buttonStart/ButtonStart.stories.ts

```diff
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import ButtonStart from ".";
+
+const meta = {
+  title: "DSN Components/ButtonStart",
+  component: ButtonStart,
+  // This component will have an automatically generated Autodocs entry: https://storybook.js.org/docs/writing-docs/autodocs
```

Comment:
do you feel all this commenting adds value - if its a documentation task, woudl the readme be better?